bin/pulsar

```diff
@@ -74,6 +110,7 @@ where command is one of:
     discovery           Run a discovery server
     proxy               Run a pulsar proxy
     websocket           Run a web socket proxy server
+    worker              Run a function worker server
```

Comment:
what about naming the command `functions-worker` to make it more explicit? Also, use plural to highlight this will be running multiple instances of multiple functions.